libbpf-tools/trace_helpers.h

```diff
@@ -53,7 +53,8 @@ partitions__get_by_dev(const struct partitions *partitions, unsigned int dev);
 const struct partition *
 partitions__get_by_name(const struct partitions *partitions, const char *name);
 
-void print_log2_hist(unsigned int *vals, int vals_size, const char *val_type);
+void print_log2_hist(unsigned int *vals, int vals_size, const char *val_type,
```

Comment:
Why not add a new API and minimize the changes ?

Reply:
I added a print of the label, which I think is necessary, just like the `print_log2_hist` function provided by BCC Python. If I add a new API, there will be a copy of the same code.

Reply:
No need for copying.
```
void print_xxx(foo);
void print_xxx_with_label(foo, label);
```
And
```
void print_xxx(foo)
{
    print_xxx_with_label(foo, NULL);
}
```

Reply:
Yes, but I need to check that "vals" is empty first, so this code needs a duplicate copy.